modules/ROOT/pages/proxy-domain-support.adoc

```diff
@@ -59,23 +61,28 @@ To configure an API proxy to use a domain:
 +
 image::proxy-domain-support-bb01e.png[]
 
-After saving the API proxy configuration, you can perform either of the following tasks:
+. After saving the API proxy configuration, perform one of the following steps:
 
 * Deploy the API proxy to the selected target (*Settings*, under *Deployment Configuration*).
 * Download the API proxy (*Actions*) and edit it in Anypoint Studio to customize your target domain before deploying it.
 
 == Configuring a Domain for the First Time
 
+When referencing a domain, Mule reads the domain Maven coordinates from the API proxy `pom.xml` file and looks for an already deployed domain with the same semantic name. For example, in the case of an API gateway domain with the version specified in the previous example, the semantic name is: `gateway-proxy-domain-1.0.0-mule-domain`.
+
+For a proxy to leverage the domain, you must ensure that the domain template is specified in the API proxy `pom.xml` as a provided dependency. 
+
 To configure a domain for the first time:
 
 . Obtain a domain artifact in one of the following ways:
 .. Download the https://anypoint.mulesoft.com/exchange/org.mule.examples/gateway-proxy-domain/[API Gateway domain template].
-.. Develop your own custom domain, using the naming convention predefined in the autogeneated proxy: +
+.. Develop your own custom domain, using the naming convention predefined in the auto-generated proxy: +
```

Comment:
autogenerated per CMS & Merriam Webster: https://www.chicagomanualofstyle.org/book/ed17/part2/ch07/psec089.html

Reply:
Changed to autogenerated! Thanks!